open-craft/xblock-group-project-v2
# OC-1373 Display friendlier message when session expiry occours

### Bug verification instructions

1. Have Apros installed with GWv2 set up. GWv2 must contain a submission stage
2. Login to Apros as a student user and go to submission stage. 
3. Upload a file. Observe dialog with: "Your deliverable has been successfully uploaded" 
4. Open new tab an go to apros in the new tab. Log out of apros. 
5. On the first tab, observe dialog with an ugly CSRF verification error (this is the issue this PR fixes)

### Fix verification instruction

1. Checkout this branch, and wait until lms restarts itself
2. Login to Apros as a student user and go to submission stage. 
3. Upload a file. Observe dialog with: "Your deliverable has been successfully uploaded (...)" 
4. Open new tab an go to apros in the new tab. Log out of apros. 
5. On the first tab, observe dialog with message: "An error occurred while uploading your file (...)  Technical details: CSRF verification failed."
6. Technically if you trigger plain 403 you'll see a very similar message, with another technical details, it was added as a precaution if CSRF view will not contain CSRF string in the future (technically CSRF violation is not a HTTP state, so I check for the response contents for 403 responses).
